kinto_client/importer.py

```diff
@@ -0,0 +1,321 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+import argparse
+import getpass
+import logging
+import os
+from six import iteritems
+
+from kinto_client import Client
+from kinto_client.exceptions import KintoException
+
+COMMAND_LOG_LEVEL = 25
+logging.addLevelName(COMMAND_LOG_LEVEL, 'COMMAND')
+logging.basicConfig(level=COMMAND_LOG_LEVEL, format="%(message)s")
+global_logger = logging.getLogger(__file__)
+
+
+def is_same_record(fields, one, two):
+    for key in fields:
+        if one.get(key) != two.get(key):
+            return False
+    return True
+
+
+class KintoImporter(object):
+    # Default configure parser values
+    all_default_parameters = False
+    remote_server = None
+    authentication = None
+    files = None
+    verbosity = None
```

Comment:
We should probably prefix these options with `use_`. It's not clear what they do just by reading the code here.